syntax = "proto3";

package multiple_rpc;

service InfoService{
    rpc GetServerInfo (User) returns (stream Address){}
    rpc SetServerInfo(Address) returns (Message){}
    rpc UnsetServerInfo(Address) returns (Message){}
}

service GreetingService{
    rpc SayHello (User) returns (Message){}
}

message User{
    string name = 1;
}

message Address{
    string address=1;
}

message Message{
    string message = 1;
}